Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -253,7 +253,8 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
         PageTiledBacking,
         ScrollContainer,
         ScrolledContents,
-        Shape
+        Shape,
+        TiledBacking
```

Comment:
We do already have `PageTiledBacking`. Any thoughts on keeping this as-is, or renaming to indicate that this should be used with caution?

`setAlwaysUseTiling()` is not possible with the current architecture, since the LayerType is set at construction time, Can we consider a larger refactor as a follow-up?

Reply:
Seems reasonable to just go with TiledBacking. Please move it up below PageTiledBacking.

Reply:
Done – thank you!